coderd/database/dbgen/dbgen.go

```diff
@@ -1056,6 +1066,14 @@ func TemplateVersion(t testing.TB, db database.Store, orig database.TemplateVers
 			}))
 		}
 
+		if hasExternalAgent.Valid {
```

Comment:
Could this just get merged with the query above rather than doing a new query? They seem very similar